lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb

```diff
@@ -125,6 +155,81 @@ def scope_type
            }
     end
 
+    if Feature.smart_v2?
+      group from: :smart_standalone_launch_stu2,
+            config: {
+              inputs: {
+                use_pkce: {
+                  default: 'true',
+                  locked: true
+                },
```

Comment:
I see PKCE getting locked here, but in locally testing I'm able to disable it.

<details>
<summary> Example Screenshot </summary>

![Screen Shot 2022-08-08 at 6 43 13 PM](https://user-images.githubusercontent.com/41651655/183527074-422d9c54-3cd4-464b-9c9d-aadf3c5e0fa1.png)
With PKCE:

```url
https://inferno.healthit.gov/reference-server/oauth/authorization?response_type=code&client_id=SAMPLE_CONFIDENTIAL_CLIENT_ID&redirect_uri=http%3A%2F%2Flocalhost%3A4567%2Finferno%2Fcustom%2Fsmart%2Fredirect&scope=launch%2Fpatient+openid+fhirUser+offline_access+patient%2FMedication.read+patient%2FAllergyIntolerance.read+patient%2FCarePlan.read+patient%2FCareTeam.read+patient%2FCondition.read+patient%2FDevice.read+patient%2FDiagnosticReport.read+patient%2FDocumentReference.read+patient%2FEncounter.read+patient%2FGoal.read+patient%2FImmunization.read+patient%2FLocation.read+patient%2FMedicationRequest.read+patient%2FObservation.read+patient%2FOrganization.read+patient%2FPatient.read+patient%2FPractitioner.read+patient%2FProcedure.read+patient%2FProvenance.read+patient%2FPractitionerRole.read&state=83d3335e-5bf3-4a9d-b776-25597e7901c6&aud=https%3A%2F%2Finferno.healthit.gov%2Freference-server%2Fr4
```

Without PKCE:

```url
https://inferno.healthit.gov/reference-server/oauth/authorization?response_type=code&client_id=SAMPLE_CONFIDENTIAL_CLIENT_ID&redirect_uri=http%3A%2F%2Flocalhost%3A4567%2Finferno%2Fcustom%2Fsmart%2Fredirect&scope=launch%2Fpatient+openid+fhirUser+offline_access+patient%2FMedication.read+patient%2FAllergyIntolerance.read+patient%2FCarePlan.read+patient%2FCareTeam.read+patient%2FCondition.read+patient%2FDevice.read+patient%2FDiagnosticReport.read+patient%2FDocumentReference.read+patient%2FEncounter.read+patient%2FGoal.read+patient%2FImmunization.read+patient%2FLocation.read+patient%2FMedicationRequest.read+patient%2FObservation.read+patient%2FOrganization.read+patient%2FPatient.read+patient%2FPractitioner.read+patient%2FProcedure.read+patient%2FProvenance.read+patient%2FPractitionerRole.read&state=4ce308a3-c202-42e1-af5c-e668eef99b14&aud=https%3A%2F%2Finferno.healthit.gov%2Freference-server%2Fr4&code_challenge=CR3ugnpBijaYsmMb5JcnGjMV1CwxAw0iLfeCUE46Y5k&code_challenge_method=S256
```
</details>

It's also unlocked in the EHR launch. It is _locked_ for the limited and public launches though.

Reply:
I only tested this after I rebased and didn't see that, but you should give it another try and see if it's still an issue for you.

Reply:
Still appearing unlocked for me

Reply:
it's locked when running directly from this repo. Weird